ext/pgsql/pgsql.c

```diff
@@ -4828,8 +4792,8 @@ PHP_PGSQL_API int php_pgsql_convert(PGconn *pg_link, const char *table_name, con
 							/* better to use PGSQLescapeLiteral since PGescapeStringConn does not handle special \ */
 							ZSTR_LEN(str) = PQescapeStringConn(pg_link, ZSTR_VAL(str), Z_STRVAL_P(val), Z_STRLEN_P(val), NULL);
 							str = zend_string_truncate(str, ZSTR_LEN(str), 0);
```

Comment:
This truncate looks unnecessary now, as add_quotes will be allocating a string of the right size anyway.